### ISSUE TITLE
overview: Usage bars lack ARIA labelling

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#15187]
On the overview page, the progress bar widgets we use lack ARIA labelling. They have `aria-labelledby` and point to an ID of an element that does not exist.

Meanwhile, the text next to the progress bars explains what's going on, so instead of `aria-labelledby`, I think we can just use `aria-hidden="true"` instead.

**Comment by KKoukiou** [comment]
This is an issue in PF itself, https://github.com/patternfly/patternfly-react/issues/5396
